README.md

```diff
@@ -92,14 +96,36 @@ If you want more information about forking you can find it
 You most likely want to make changes to the files in the folder `en`. That's
 where all the real documentation text is located.
 
-#### Testing your changes
+#### Testing your changes (on your local machine)
 
 ```sh
 pipenv install -r requirements.txt
 pipenv run mkdocs serve
 ```
 
-The local doc will be available at http://localhost:8000
+The local doc will be available at http://localhost:8000.
+
+To test your work against your pre-commit hook:
+
+    pipenv install pre-commit
+    pre-commit run --all-files
```

Comment:
```suggestion
#### Contribute
Before commiting, install [pre-commit](https://pre-commit.com/) to auto-format your contributions.

```
pip install pre-commit
pre-commit install
```

Reply:
Thank you very much, committed your suggestion, amended to pip install --user to minimize footprint on system-wide Python.